tests/framework/microvm.py

```diff
@@ -297,6 +299,17 @@ def setup(self):
         os.makedirs(self._kernel_path, exist_ok=True)
         os.makedirs(self._fsfiles_path, exist_ok=True)
 
+    def init_snapshot(self):
```

Comment:
A name like `init_snapshotting_api` is less vague.

Reply:
done